### ISSUE TITLE
stg-backups going to wrong place

### ISSUE THREAD

**Issue by PaulBags** [Drive4ik/simple-tab-groups#1112]
**Describe the bug**
I found stg-backup folders on a usb drive. There's no way I would have copied them there because it wasn't even my drive, so this is actually a security risk as well.

**To Reproduce**
I have no idea how it happened. STG saved it's backup folder somewhere it shouldn't have and shouldn't have been able to.

**Expected behavior**
Files should be saved with firefox, and failing that not outside of a users own files.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS and version: Windows 10
 - Firefox version: 110.0.1 (according to the folder name)
 - Simple Tab Groups version: No idea. I froze on Firefox 110.0.1 for months before giving in recently and just putting up with the worthless and invasive puzzle icon Mozilla decided we had to have, it could have been any STG version from latest to a year ago.

**Additional context**
Sorry I wish I could help more, all I know is this isn't OK and STG should find another way of dealing with backups than, apparently, assuming Firefox restricted extensions environment will do sane things.

**Comment by H-Sapien** [comment]
Yes, I had the same surprise trouble when coming back to another computer and another profile.
All my STG-backups groups had gone!
Curiously, I found them, the regular (Update Ff) saves, not in my local account's Download folder but in Public's.
Likewise, I have no idea how they got there, I'm still searching for a way to specify where to put the periodic saved sets. There's a set at each update of Firefox, and the list of folders is so long it's getting in the way.
'STG saved its backup folder somewhere it shouldn't have and shouldn't have been able to.'
Love the idea, but... 
H Sap.

**Comment by jonfre** [comment]
I'm using version 5.2 on Linux. 

Setting the backup Folder name to 
`/home/users/bob/backup/SimpleTabGroups`
creates backups in
`/home/users/bob/Downloads/homeusersbobbackupSimpleTabGroups`

**Comment by aamirazad** [comment]
I'm pretty sure stg backups files to a folder in whatever downloads folder is set by firefox. 
![image](https://github.com/Drive4ik/simple-tab-groups/assets/82281117/d319f5b8-a8d1-4596-b3f4-b3596c53e99f)
I don't they can change this behavior. Going to stg settings and under backups, you can click open backup folder to show where backups are going

**Comment by H-Sapien** [comment]
Okay, that makes it clear, thanks Aamir.
The download folder is set by Firefox - and the STG backups will accumulate in each of their own folders within that download-specified folder.
The list is getting very long (added to weekly (see below*)) and so the S...s in alphabetical order are becoming a nuisance - so I expect, if I'm going to keep them, the best thing to do is to create an STG folder, and move all the STG-backups-FF... into that. There doesn't seem to be any dependency by STG on their being there, so it shouldn't hurt. I'm thinking that they're there for reversion, should I need to bring sets of tabs back into use, so I can just pick them up from where I've put them.
*I do notice that the backup saves are not weekly but by Firefox version. For each change there seems to be a backup made. I'm trying to think why. _[Thinks]_
